Examples/Python/python/acts/examples/simulation.py

```diff
@@ -555,13 +559,19 @@ def addGeant4(
     else:
         particles_selected = "particles_input"
 
-    g4detector = DDG4DetectorConstruction(geometryService)
+    if g4detectorConstruction is None:
+        if detector is None:
+            raise ArgumentError("detector not given")
+        g4detectorConstruction = getG4DetectorContruction(detector)
+
     g4conf = geant4SimulationConfig(
         level=customLogLevel(),
-        detector=g4detector,
+        detector=g4detectorConstruction,
         inputParticles="particles_input",
         trackingGeometry=trackingGeometry,
         magneticField=field,
+        volumeMappings=[],
+        materialMappings=["G4_Si"],
```

Comment:
In principle I'm fine with this, but this means that this material is hardcoded now... Do you have checked if the ODD still works with this?
The best solution would be to couple the material mapping information the detectorconstruction object, but it would be maybe also sufficient to add more entries to the default, e.g. `["G4_Si", "Silicon", "Si"]` I guess (`"Silicon"` is the default entry on the C++ side, which gets overwritten by this).

Reply:
oh yeah that should not be merged. I think it would be good to expose the list as you did. but I will use "normal" silicon in the G4 construction